package.json

```diff
@@ -25,7 +25,7 @@
     "testci": "jest --maxWorkers=2",
     "security_audit": "npx audit-ci --config audit-ci.json",
     "int-test": "cypress run --config video=false",
-    "int-test-ui": "cypress open",
+    "int-test-ui": "cypress open --e2e --browser electron",
```

Comment:
why `electron` here instead of `chrome`?

Reply:
I do like the `e2e` setting though - hadn't realised that we could do that

Reply:
I thought it provides a bit of browser variation as we use chrome normally .. although I just remembered the headless run uses electron anyway, so perhaps not! Will change next time.